src/components/Disputes/actions/DisputeAppeal.js

```diff
@@ -24,8 +24,22 @@ function DisputeAppeal({ onRequestAppeal, confirm }) {
       >
         {actionLabel}
       </Button>
-      <Info mode="description">
-        Anyone holding the required appeal deposit can trigger this action.
+      <Info>
+        <strong>Anyone</strong> can{' '}
+        <strong>
+          lock DAI as collateral to{' '}
+          {confirm ? 'confirm an appeal' : 'propose an appeal'}{' '}
```

Comment:
Bit of a nitpick, but "propose an appeal" sounds awkward in English—if there's no taker on the other side, you automatically win. Is this what we use in the user guide and etc?

What do you think about "begin" or "initiate" or "advocate for", as they're more firm than "propose"?

Reply:
Yes i think it makes sense, as you well say, proposing is a weak word and doesn't reflect properly the intentions of the appealer.